pydata_sphinx_theme/static/css/theme.css

```diff
@@ -23,8 +23,12 @@
   /*****************************************************************************
   * Font family
   **/
-  --pst-font-family-base: 'Lato', sans-serif;
-  --pst-font-family-heading: 'Open Sans', sans-serif;
+  /* These are adapted from https://systemfontstack.com/ */
+  --pst-font-family-base: -apple-system, BlinkMacSystemFont, Segoe UI, "Helvetica Neue",
+    Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol;
+  --pst-font-family-heading: var(--font-family-base);
+  --pst-font-family-monospace: "SFMono-Regular", Menlo, Consolas, Monaco,
+    Liberation Mono, Lucida Console, monospace;
```

Comment:
i don't know enough SCSS... do Liberation and Lucida need to be quoted in this dialect?

Reply:
Also, one thing we added over on [lab](https://github.com/jupyterlab/jupyterlab/blob/master/packages/theme-dark-extension/style/variables.css#L185
): we put the _actual_ system fallback chain into a _dedicated_ variable, and then use that as the default of the other things, so it's available even if a theme wishes to overload it (but have a nice fallback). so with spaces from above:
```scss
:root {
  --pst-font-family-base-system: -apple-system, BlinkMacSystemFont, "Segoe UI", "Helvetica Neue",
    Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
  --pst-font-family-monospace-system: "SFMono-Regular", Menlo, Consolas, Monaco,
    "Liberation Mono", "Lucida Console", monospace;

  --pst-font-family-base: var(--pst-font-family-base-system);
  --pst-font-family-heading: var(--font-family-base);
  --pst-font-family-monospace: var(--pst-font-family-monospace-system);
}
```

A custom CSS could then:

```css
:root {
  --pst-font-family-base: "Super Duper Sans", var(--pst-font-family-fallback-base);
}
```

Reply:
> i don't know enough SCSS... do Liberation and Lucida need to be quoted in this dialect?

This file is actually plain CSS, not SCSS. 
But good question. It seems that those quotes are optional but recommened: https://stackoverflow.com/questions/13751412/why-would-font-names-need-quotes

Good idea for the separate variable for system default.